backend/routes/restaurant.js

```diff
@@ -8,6 +8,7 @@ const connection = require('../database');
 const router = express.Router();
 router.use(bodyParser.urlencoded({ extended: true }));
 
+// Get Specific restaurant name
```

Comment:
Currently the database stores the name, but later it will store other info as well, such as banner image url.

Reply:
Updated comment to reflect that.